parsers/CA_SK.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import arrow
+import requests
+
+from parsers.lib.exceptions import ParserException
+
+CONSUMPTION_URL = 'https://www.saskpower.com/ignitionapi/Content/GetNetLoad'
+SOURCE = 'saskpower.com'
+ZONE_KEY = 'CA-SK'
+
+
+def fetch_consumption(zone_key=ZONE_KEY, session=None, target_datetime=None, logger=None):
+    if target_datetime:
+        raise NotImplementedError('This parser is not yet able to parse past dates')
+
+    request = session or requests.session()
+    response = request.get(url=CONSUMPTION_URL)
+    if response.ok:
+        decoded_content = response.content.decode(response.encoding)
+        consumption = float(decoded_content.strip('"'))
+        date_header = response.headers.get('date')
+        consumption_date = arrow.get(date_header, 'ddd, DD MMM YYYY HH:mm:ss ZZZ').datetime
```

Comment:
I realize that the date of the HTTP response isn't really a great value to be used for, "the date of the data." I'm going to leave my computer up and polling for a while to sample how often this number changes. From what I've seen during development it's fairly frequently.

However, even if those results show it's fresh, I don't know how we could programmatically return a better date to describe when the consumption value was observed.

Reply:
As promised, I set up a cron job locally to poll `CA_SK.fetch_consumption(...)` once per minute over the past day. I can share the results if requested, but this graph from a spreadsheet I quickly put together summarizes things well.

![saskpower_consumption_2021-12-10](https://user-images.githubusercontent.com/3433964/145642325-f7f68fa3-0805-4b1a-b3f0-e1d8d563899d.png)

The value is updated every ~5 minutes. It's a little jumpy but seems reasonable.